anecdote/assets/logs/anecdote_tests.yaml

```diff
@@ -0,0 +1,154 @@
+id: anecdote
+tests:
+  - sample: >-
+      {
+      	"id": "AgAAAZB5YmFzhzsIeQAAAAAAAAAYAAAAAEFaQjVZbUZ6QUFEbVVJd21ZUWhyMVFDdQAAACQAAAAAMDE5MDc5NjItNjE3My00NjUzLWFiNTgtZTRkNWI3ZmE2ZjU0",
+      	"content": {
+      		"timestamp": "2024-07-03T16:16:49.011Z",
+      		"tags": [
```

Comment:
I am not very clear what these tags are about? Is the goal for these tags to injected in datadog tag logs? As of now they will only be available in the Event attributes.

<img width="456" alt="image" src="https://github.com/user-attachments/assets/a07e8779-231c-4979-889b-5d4bf2697804">